docs/integrations/mlflow.rst

```diff
@@ -3,54 +3,91 @@
 Neptune-MLflow Integration
 ==========================
 
-|neptune-mlflow| is an open source project curated by Neptune team that enables |mlflow| experiment runs to be hosted in Neptune.
+|Run on Colab|
+
+What will you get with this integration?
+----------------------------------------
+
+|mlflow-tour|
+
+|neptune-mlflow| is an open source project curated by Neptune team that enables |mlflow| experiment runs to be hosted on Neptune.
 
 The integration lets you enjoy the best of both worlds: the tracking and reproducibility of MLflow with the organization and collaboration of Neptune.
 
+.. note::
+
+    This integration is tested with ``mlflow==1.12.1``
+	
 .. image:: ../_static/images/integrations/mlflow_neptuneml.png
    :target: ../_static/images/integrations/mlflow_neptuneml.png
    :alt: organize MLflow experiments in Neptune
 
-Check the example project in Neptune: |mlflow-integration|.
+.. _mlflow-quickstart:
 
-Installation
-------------
+Quickstart
+----------
 
-.. code-block:: bash
+This quickstart will show you how to:
+
+* Install the necessary neptune packages
+* Sync your MLruns with Neptune.
```

Comment:
I think sync is the word that may not be easily understandable to a new person. 
Perhaps we should use convert MLruns directory of experiment runs into Neptune experiments?